s3/sc-s3-synapse-ra.yaml

```diff
@@ -70,6 +70,14 @@ Resources:
       OwnershipControls:
         Rules:
           - ObjectOwnership: BucketOwnerPreferred
+      LifecycleConfiguration:
+        Rules:
+          - Id: IntelligentTieringRule
+            Prefix: IntTier
```

Comment:
The Prefix is used to identifies which objects the rule applies to, in particular it would be an object key prefix so I do not think we need this.